go/api/standalone_client.go

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+package api
+
+import "C"
+
+// RedisClient is a client used for connection to standalone Redis servers.
+type RedisClient struct {
+	*baseClient
+}
+
+// NewRedisClient creates a Redis client in standalone mode using the given [RedisClientConfiguration].
+func NewRedisClient(config *RedisClientConfiguration) (*RedisClient, error) {
```

Comment:
In Java this is called CreateClient, but in Go the idiom is that a function called NewRedisClient should be provided if users aren't supposed to initialize a RedisClient struct directly. Also, the Go docs normally will document exported symbols alphabetically but will place New functions immediately underneath the document for the associated struct. We lose this clarity if we call this CreateClient